backend/controllers/userDetails.js

```diff
@@ -22,7 +22,7 @@ const userDetailsFactory = (
 
     const searchTitle = req.session.searchTitle ? req.session.searchTitle : defaultSearchTitle
     const searchUrl = req.session.searchUrl ? req.session.searchUrl : defaultSearchUrl
-    const searchResultsUrl = req.session.searchResultsUrl ? req.session.searchResultsUrl : `${searchUrl}/results`
```

Comment:
Removing `/results` part of default search results url - produces wrong url for the dps search and an unuseful page for the external users search (since no filters from the session so you just get everything...)